controllers/core/openstackcontrolplane_controller.go

```diff
@@ -213,35 +223,35 @@ func (r *OpenStackControlPlaneReconciler) reconcileNormal(ctx context.Context, i
 		return ctrlResult, nil
 	}
 
-	ctrlResult, err = openstack.ReconcileCinder(ctx, instance, helper)
+	ctrlResult, err = openstack.ReconcileOVN(ctx, instance, helper)
 	if err != nil {
 		return ctrl.Result{}, err
 	} else if (ctrlResult != ctrl.Result{}) {
 		return ctrlResult, nil
 	}
 
-	ctrlResult, err = openstack.ReconcileOVN(ctx, instance, helper)
+	ctrlResult, err = openstack.ReconcileOVS(ctx, instance, helper)
 	if err != nil {
 		return ctrl.Result{}, err
 	} else if (ctrlResult != ctrl.Result{}) {
 		return ctrlResult, nil
 	}
 
-	ctrlResult, err = openstack.ReconcileOVS(ctx, instance, helper)
+	ctrlResult, err = openstack.ReconcileNeutron(ctx, instance, helper)
 	if err != nil {
 		return ctrl.Result{}, err
 	} else if (ctrlResult != ctrl.Result{}) {
 		return ctrlResult, nil
 	}
 
-	ctrlResult, err = openstack.ReconcileNeutron(ctx, instance, helper)
+	ctrlResult, err = openstack.ReconcileNova(ctx, instance, helper)
 	if err != nil {
 		return ctrl.Result{}, err
 	} else if (ctrlResult != ctrl.Result{}) {
 		return ctrlResult, nil
 	}
 
-	ctrlResult, err = openstack.ReconcileNova(ctx, instance, helper)
+	ctrlResult, err = openstack.ReconcileCinder(ctx, instance, helper)
```

Comment:
Is there a specific reason why cinder is moved to the last?

Reply:
In the earlier patchsets, I had added handling for it being disabled in this file. It's position is just a remnant of that previous patchset, I'll move it back in my next revision